enricher/license/license.go

```diff
@@ -104,16 +102,28 @@ func (e *Enricher) Enrich(ctx context.Context, _ *enricher.ScanInput, inv *inven
 		if !ok {
 			continue
 		}
-		queries = append(queries, versionQuery(ecoSystem, pkg.Name, pkg.Version))
+		queries[i] = versionQuery(ecoSystem, pkg.Name, pkg.Version)
 	}
 
 	licenses, err := e.makeVersionRequest(ctx, queries)
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to get version information %w", err)
 	}
 
 	for i, license := range licenses {
-		inv.Packages[i].License = license
+		// use license information from deps.dev if available (preferred source of truth)
```

Comment:
I'm not sure if this (preferring deps.dev as source of truth over local extraction) is what we actually want, @alowayed thoughts? It's probably fine to leave it for now and adjust later though